README.md

```diff
@@ -1 +1,181 @@
-# heli-hire-captsone-frontend
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>HeliHire 🏠️🛠️</b></h1>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [ HandyMate 🏠️🛠️ ](#-HandyMate-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# HeliHire 🏠️🛠️ <a name="about-project"></a>
+
+<b>HeliHire</b> is a website where users can book a Helicopter, cancel a booking. The user can also see the details of a Helicopter, ratings of the service.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### App Screenshot
+
+![Screenshot 2023-10-12 at 14 01 17 (8)](https://github.com/emmanuelsteven/heli-hire-captsone-frontend/assets/96125730/ab9b0592-9e3e-4201-b0b2-6a22b4a5278d)
+
+
+### Back-end Repository
+- Here is the link of the [Back-End](https://github.com/emmanuelsteven/heli-hire-captone-backend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone this repository using the GitHub link provided below.
+
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/emmanuelsteven/heli-hire-captsone-frontend.git
+```
+
+### Usage
+
+
+Kindly modify the files as needed.
+
+In the project directory, you can run:
+```
+$ npm start
+```
+Runs the app in the development mode.\
+Open [http://localhost:5173/] to view it in your browser.
+
+The page will reload when you make changes.
+
+You may also see any lint errors in the console.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+$ npm test
+```
```

Comment:
- [ ] I couldn't find any frontend test in your project.  Kindly either add some tests or remove the section. 👍